Robot/neopixel-leds/src/test/java/org/usfirst/frc/team4999/tools/PixelComparator.java

```diff
@@ -0,0 +1,75 @@
+package org.usfirst.frc.team4999.tools;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Vector;
+
+import org.usfirst.frc.team4999.tools.gui.DiffShower;
+import org.usfirst.frc.team4999.lights.Color;
+
+import static org.junit.Assert.fail;
+
+public class PixelComparator implements BufferDisplay.BufferUpdateListener {
+
+    private Vector<Color[]> displayHistory;
+
+    public PixelComparator() {
+        displayHistory = new Vector<Color[]>();
+    }
+
+    @Override
+    public void onBufferUpdated(Color[] buffer) {
+        
+        displayHistory.add(buffer);
+    }
+
+    public void writeToFile(String name) {
+        try {
+            AnimationFileManager.saveFile(displayHistory, name);
+        } catch (IOException e) {
+            e.printStackTrace();
+            throw new RuntimeException(String.format("Failed to save %s", name));
+        }
+    }
+    public void compareToFile(String filename) {
```

Comment:
It's an overload, so that compareToFile defaults to headless operation. That way, you don't have to specify `compareToFile(filename, false)` every time, you can just write `compareToFile(filename)` and it will default to false.
Also, since I'm adding the headless argument to a preexisting method, I this stub leaves backwards compatibility with preexisting tests.